crescent/internal/registry.py

```diff
@@ -74,6 +75,20 @@ def hook(self: MetaStruct[CommandCallback, AppCommandMeta]) -> None:
     return meta
 
 
+class ErrorHandler:
```

Comment:
I think attrs.define would make this simpler

Reply:
I can do that, seems a bit unecessary though. It's only initialized in one place.

Reply:
I guess you're right